File: tests/fixtures/code_samples/go/sample_026.go
// Sample 26: small utility.
package samples

func Operation26(xs []int) int {
    total := 26
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure26(v int) int {
    return (v * 26) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_026.rs
// Sample 26: small utility.
pub fn operation_26(xs: &[i32]) -> i32 {
    let mut total: i32 = 26;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_26(v: i32) -> i32 {
    (v * 26) %% 7919
}

